<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>551</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>166</x>
       <y>30</y>
       <width>391</width>
       <height>91</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>700</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../img/logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>571</width>
       <height>301</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>700</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0.511364, y1:1, x2:0.500227, y2:0, stop:0 rgba(138, 138, 138, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="applications">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>700</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(182, 182, 182);
background-color: qlineargradient(spread:pad, x1:0.511364, y1:1, x2:0.489, y2:0, stop:0 rgba(86, 86, 86, 255), stop:1 rgba(255, 255, 255, 255));
background-color: qlineargradient(spread:pad, x1:0.511364, y1:1, x2:0.489, y2:0, stop:0 rgba(86, 86, 86, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string>Applications</string>
      </property>
     </widget>
     <widget class="QPushButton" name="interviews">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>700</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(182, 182, 182);
background-color: qlineargradient(spread:pad, x1:0.511364, y1:1, x2:0.489, y2:0, stop:0 rgba(86, 86, 86, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string>Interviews</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mentor_meet">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>700</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(182, 182, 182);
background-color: qlineargradient(spread:pad, x1:0.511364, y1:1, x2:0.489, y2:0, stop:0 rgba(86, 86, 86, 255), stop:1 rgba(255, 255, 255, 255));
background-color: qlineargradient(spread:pad, x1:0.511364, y1:1, x2:0.54, y2:0, stop:0 rgba(82, 82, 82, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string>Mentor Meeting</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>220</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>700</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(182, 182, 182);
background-color: qlineargradient(spread:pad, x1:0.511364, y1:1, x2:0.489, y2:0, stop:0 rgba(86, 86, 86, 255), stop:1 rgba(255, 255, 255, 255));
color:rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="main_menu">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>150</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(182, 182, 182);
background-color: qlineargradient(spread:pad, x1:0.511364, y1:1, x2:0.489, y2:0, stop:0 rgba(86, 86, 86, 255), stop:1 rgba(255, 255, 255, 255));
color:rgb(0, 0, 0)</string>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="admin_menu">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>220</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>700</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(182, 182, 182);
background-color: qlineargradient(spread:pad, x1:0.511364, y1:1, x2:0.489, y2:0, stop:0 rgba(86, 86, 86, 255), stop:1 rgba(255, 255, 255, 255));
color:rgb(0, 0, 0)
</string>
      </property>
      <property name="text">
       <string>Admin Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>700</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(218, 30, 60);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>-10</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>700</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;CRM - Preference Admin Menu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
